skeleton/aws/modules/ecs/main.tf

```diff
@@ -42,34 +55,57 @@ locals {
       }
     ]
   }
+
+  # Required IAM permissions from
+  # https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-auto-scaling.html#auto-scaling-IAM
+  ecs_service_scaling_policy = {
+    Version = "2012-10-17",
+    Statement = [
+      {
+        Effect = "Allow",
+        Action = [
+          "application-autoscaling:*",
+          "ecs:DescribeServices",
+          "ecs:UpdateService",
+          "cloudwatch:DescribeAlarms",
+          "cloudwatch:PutMetricAlarm",
+          "cloudwatch:DeleteAlarms",
+          "cloudwatch:DescribeAlarmHistory",
+          "cloudwatch:DescribeAlarmsForMetric",
+          "cloudwatch:GetMetricStatistics",
+          "cloudwatch:ListMetrics",
+          "cloudwatch:DisableAlarmActions",
+          "cloudwatch:EnableAlarmActions",
+          "iam:CreateServiceLinkedRole",
+          "sns:CreateTopic",
+          "sns:Subscribe",
+          "sns:Get*",
+          "sns:List*"
+        ],
+        Resource = "*"
+      }
+    ]
+  }
 }
 
 # Current task definition on AWS including deployments outside terraform (e.g. CI deployments)
 data "aws_ecs_task_definition" "task" {
   task_definition = aws_ecs_task_definition.main.family
 }
 
-data "aws_iam_policy_document" "ecs_task_execution_role" {
-  version = "2012-10-17"
-  statement {
-    sid     = ""
-    effect  = "Allow"
-    actions = ["sts:AssumeRole"]
-
-    principals {
-      type        = "Service"
-      identifiers = ["ecs-tasks.amazonaws.com"]
-    }
-  }
-}
-
 resource "aws_iam_policy" "ecs_task_execution_ssm" {
+  name   = "${var.namespace}-ECSTaskExecutionAccessSSMPolicy"
   policy = jsonencode(local.ecs_task_execution_ssm_policy)
 }
 
+resource "aws_iam_policy" "ecs_task_excution_service_scaling" {
```

Comment:
To avoid errors in the develop branch after merging different branches, we can disregard this line beforehand. There is only one warning for this particular branch.
```suggestion
# tfsec:ignore:aws-iam-no-policy-wildcards
resource "aws_iam_policy" "ecs_task_excution_service_scaling" {
```

Reply:
thank you. Updated in ba44f31.